EIPS/eip-2381.md

```diff
@@ -0,0 +1,146 @@
+---
+eip: 2381
+title: ENS support for ERC 721 TokenID
+author: Oisin Kyne (@OisinKyne) <oisin@kyne.eu>
+discussions-to: https://github.com/ethereum/EIPs/issues/2381
+status: Draft
+type: Meta
+created: 2019-11-15
+requires (*optional): 137, 165, 721
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+
+_TL;DR_: This EIP makes ERC721 Non-Fungible Tokens addressable by the Ethereum Name Service by adding a new resolver profile, `uint256 tokenID()`.
```

Comment:
```suggestion
This EIP makes [ERC-721](./eip-721.md) Non-Fungible Tokens addressable by the Ethereum Name Service by adding a new resolver profile, `uint256 tokenID()`.
```
The first reference to any EIP in an EIP should be a link to that EIP.  Also, the correct formatting for EIP/ERCs is `EIP-XXX` (note the hyphen).  That needs to be fixed throughout this document.

Reply:
Fixed throughout. Used the FQDN (e.g. https://eips.ethereum.org/EIPS/eip-20) rather than a relative link within the repo if that's acceptable. Can change.

Reply:
They do need to be relative links.  This ensures that they work when viewing in GitHub, on EIPs repository, in web archive, on a mirror, etc.